### DISCUSSION TITLE
Allowing colour customisation of the line number column.

### DISCUSSION THREAD

**Discussion by HubKing** [tsujan/FeatherPad#790] [Ideas]
I am using dark theme, and the line number column is light mode (black on white). I could not find a way to change it. I wish I could set colours to make it dark.

![image](https://github.com/tsujan/FeatherPad/assets/10900489/7fd23254-aca9-44e7-a1ba-ab4688972ea2)

**Comment by tsujan** [comment]
It's made light intentionally, to have a high contrast with the view's dark color scheme, as it's dark when the view is light.


I'm afraid I don't have a plan to make it configurable. To keep the app lightweight and manageable, I avoid options that aren't justified functionally.

**Comment by ghost** [comment]
Anyone using a dark theme will say having it actually be DARK is a functional need. I'm sure adding two extra optional color definitions (dark, contrast dark, light, contrast light) isn't going to bloat the code base. That white strip is brutal and jarring, never mind for someone with one of those high NIT gaming monitors...talk about staring into the sun. Spend any decent amount of time in Featherpad and then walk around the rest of the day with a vertical racing stripe in you vision.

**Comment by twonoise** [comment]
Please allow me for my two cents.

With my respect to author's colors decision. Sure, there was a reason for it. And I am thankful to @tsujan for sharing this software with us.

At other side, while this is more or less individual, ambient light condition dependent, and time spent dependent, but, such a high contrast transients can be not good for eyes health.

If there is no dark mode for particular software, there is **Compiz Color Filter**, which negates window but keep colors (red remains red, but not aqua, like with regular rgb inversion), and this filter is a must have for science desktop.

But, CCF, or any other tool I know, can't fight with high contrast areas within same window. So, to avoid, or to customize, these areas, is fundamentally more important for eyes health, than dark mode.

Thanks for reading & understanding this. <3
